concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -745,6 +749,7 @@ jobs:
                                         ./paas-bootstrap/manifests/runtime-config/addons-meta/collectd.yml"
                 fi
 
+                # shellcheck disable=SC2086
```

Comment:
Why are we disabling the double quote check here?

Reply:
because in this instance, we want this to be split into multiple args to the `build_manifest.sh` script. This variable is set to the list of manifest files to be merged into a single output.

Reply:
Makes sense, thanks.